src/main/java/io/hops/metadata/ndb/ClusterjConnector.java

```diff
@@ -364,7 +365,8 @@ private boolean formatHDFS(boolean transactional) throws StorageException{
         HashBucketDataAccess.class, StorageDataAccess.class,
         AceDataAccess.class, RetryCacheEntryDataAccess.class, CacheDirectiveDataAccess.class,
         CachePoolDataAccess.class, CachedBlockDataAccess.class,
-        ActiveBlockReportsDataAccess.class, XAttrDataAccess.class, EncryptionZoneDataAccess.class);
+        ActiveBlockReportsDataAccess.class, XAttrDataAccess.class, EncryptionZoneDataAccess.class,
+        FileProvenanceDataAccess.class, FileProvXAttrBufferDataAccess.class, AppProvenanceDataAccess.class);
```

Comment:
can you move the AppProvenanceDataAccess to the formatYarn function

Reply:
fixed